docs/Contributing.asciidoc

```diff
@@ -415,13 +415,17 @@ as part of openSUSE Tumbleweed.
    push directly) locally, e.g. `osc co devel:openQA/openQA`.
 2. Get the script `update-cache.sh` from the main git repository with
    `osc service ra`
-3. Run `bash *update-cache.sh` inside the repository folder. Follow the log
-   checking no download errors occurred.
-4. Do a sanity check on the generated `cache.txz`. It usually should not be
+3. If you changed a url or added a new asset, you have to run the steps from
+   `update-cache.sh` manually, and instead of running
+   `./tools/generate-packed-assets` from the upstream git, you have to run it
+   on the git checkout that contains the change.
+4. If there are no changes or new urls, run `bash *update-cache.sh` inside the
+   repository folder. Follow the log checking no download errors occurred.
+5. Do a sanity check on the generated `cache.txz`. It usually should not be
```

Comment:
```suggestion
5. Check the generated `cache.txz`. It usually should not be
```

I would prefer the ableist term "sanity check" not be used.

Reply:
done

Reply:
and actually pushed now...